Classes/Emogrifier.php

```diff
@@ -1501,4 +1505,34 @@ private function getNodesToExclude(\DOMXPath $xPath)
 
         return $excludedNodes;
     }
+
+    /**
+     * Handle warnings in process() method
+     *
+     * @param int $errNo
+     * @param string $errStr
+     * @param string $errFile
+     * @param int $errLine
+     * @param array $errContext
+     *
+     * @return bool
+     *
+     * @throws \InvalidArgumentException
+     */
+    public function errorHandler($errNo, $errStr, $errFile, $errLine, $errContext)
+    {
+        if ($errNo === E_WARNING && isset($errContext['cssRule']['selector'])) {
```

Comment:
not really :) but
1. code sniffer trigger error about not used variable
2. errorHandler will be called even for E_ERRORS